### PR TITLE
Documentation update and microfixes

### PR DESCRIPTION
**Pull request by roman-khimov**
### Problem

Outdated/missing documentation.

### Solution

Fix some things here.

### CODE REVIEW

File: docs/rpc.md [hunk 1]
```diff
@@ -97,23 +103,35 @@ and we're not accepting issues related to them.
...
 Some additional extensions are implemented as a part of this RPC server.
 
+#### `getblocksysfee` call
+
+This method returns cumulative system fee for all transactions included in a
+block. It can be removed in future versions, but at the moment you can use it
+to see how much GAS is burned with particular block (because system fees are
+burned).
+
+#### `submitnotaryrequest` call
```
Comment (AnnaShaleva) [inline docs/rpc.md:130]: Add `getblocksysfee` and `submitnotaryrequest` to the `Supported Methods` list?

File: docs/rpc.md [hunk 2]
```diff
@@ -97,23 +103,35 @@ and we're not accepting issues related to them.
...
+
+This method can be used on P2P Notary enabled networks to submit new notary
+payloads to be relayed from RPC to P2P.
```
Comment (AnnaShaleva) [inline docs/rpc.md:133]: Maybe it's worth to mention that `P2PSigExtensions` should be enabled in the server config.

File: pkg/interop/native/roles/roles.go [hunk 3]
```diff
@@ -20,6 +20,7 @@ type Role byte
 const (
 	StateValidator Role = 4
 	Oracle         Role = 8
+	NeoFSAlphabet  Role = 16
```
Comment (AnnaShaleva) [inline pkg/interop/native/roles/roles.go:23]: We need to add it to `TestRoleManagementRole` also.

**Review by AnnaShaleva — CHANGES_REQUESTED** [review]

File: docs/cli.md [hunk 4]
```diff
@@ -1,183 +1,348 @@
...
+allows to dump chain into file from the database (when node is stopped) or to
+import blocks from file into the database (also when node is stopped). Use
+`db` command for that.
 
-`./bin/neo-go node --config-path /user/yourConfigPath`
+## Smart contracts
 
-## Configuration
+Use `contract` command to create/compile/deploy/invoke/debug smart contracts,
+see [compiler documentation](compiler.md).
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply docs/cli.md:65]: `cli.md` is in `docs` already, so I think it should work this way.

File: docs/rpc.md (hunk 1 above)
Comment (roman-khimov) [reply docs/rpc.md:130]: They're not standard, thus they're listed as extensions.

File: docs/rpc.md (hunk 2 above)
Comment (roman-khimov) [reply docs/rpc.md:133]: `on P2P Notary enabled networks`, that's enough for now, how to configure this feature is more of #1848 problem.

File: README.md [hunk 5]
```diff
@@ -36,60 +36,38 @@ Neo 3 code (0.90.0 being compatible with Neo 3 preview2).
...
-```
-
-Which will start a node on `privnet` and expose the nodes port `20332` and `20331` for the `JSON-RPC` server.
-
+NeoGo is distributed as a single binary that includes all the functionality
+provided (but smart contract compiler requires Go compiler to operate). You
+can grab it from [releases
+page](https://github.com/nspcc-dev/neo-go/releases), use a Docker image (see
+[Docker Hub](https://hub.docker.com/r/nspccdev/neo-go) for various releases of
+NeoGo, `:latest` points to the latest release) or build yourself.
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply README.md:44]: This is a Neo N3 branch, so maybe we can leave it as is (Neo Legacy has a README of its own in `master-2.x`).

File: pkg/interop/native/roles/roles.go (hunk 3 above)
Comment (roman-khimov) [reply pkg/interop/native/roles/roles.go:23]: Fixed!

File: README.md [hunk 6]
```diff
@@ -100,7 +78,17 @@ Available network flags:
...
-#### Importing mainnet/testnet dump files
+### Docker
+
+By default the `CMD` is set to run a node on `privnet`, so to do this simply run:
+
+```bash
+docker run -d --name neo-go -p 20332:20332 -p 20331:20331 nspccdev/neo-go
+```
+
+Which will start a node on `privnet` and expose the nodes port `20332` and `20331` for the `JSON-RPC` server.
```
Comment (fyrchik) [inline README.md:89]: A bit hard to parse, seems like both ports are used for JSON RPC. May be smth like this?

`20332` for P2P messages and `20331` for the JSON-RPC ?

File: docs/cli.md [hunk 7]
```diff
@@ -1,183 +1,348 @@
...
-make run
-```
+### Configuration
+
+All config files are located in `./config` and they are differentiated according to the network type:
+- `protocol.mainnet.yml` belongs to `--mainnet` network mode (`-m` short option)
+- `protocol.privnet.yml` belongs to `--privnet` network mode (`-p` short
+  option) and is used by default
+- `protocol.testnet.yml` belongs to `--testnet` network mode (`-t` short option)
+- `protocol.unit_testnet.yml` used by unit tests
```
Comment (fyrchik) [inline docs/cli.md:29]: is used

File: docs/cli.md [hunk 8]
```diff
@@ -1,183 +1,348 @@
...
-- `--privnet, -p`
-- `--testnet, -t`
+By default the node will run in foreground using current standard output for
+logging.
 
-#### Configuration path
+### DB import/exports
 
-If you want to use some specific configuration path you have to use `--config-path` flag
+Node operates using some database as a backend to store blockchain data. NeoGo
```
Comment (fyrchik) [inline docs/cli.md:57]: Add some notes about db settings in config (like path is `./chains` by default)?

Comment (roman-khimov) [reply docs/cli.md:57]: #1849

File: docs/cli.md (hunk 7 above)
Comment (roman-khimov) [reply docs/cli.md]: Fixed.

File: README.md (hunk 6 above)
Comment (roman-khimov) [reply README.md]: Fixed.

**Review by AnnaShaleva — APPROVED** [review]
